airbyte-webapp/src/views/Connection/ConnectionForm/components/NonBreakingChangesPreferenceField.module.scss

```diff
@@ -9,12 +9,6 @@
   gap: variables.$spacing-md;
 }
 
-.connectorLabel {
-  max-width: 328px;
-  margin-right: variables.$spacing-xl;
-  vertical-align: top;
-}
```

Comment:
Probably copied from the Schedule field. That also needs to be cleaned up but I omitted it here because of the scope of the PR.